packages/core/components/ChoiceGroup/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChoiceGroup, IChoiceGroupOption } from "@fluentui/react";
+import * as React from "react";
+
+import styles from "./ChoiceGroup.module.css";
+
+interface Props {
+    className?: string;
+    disabled?: boolean;
+    onChange: (
+        ev?: React.FormEvent<HTMLElement | HTMLInputElement>,
+        option?: IChoiceGroupOption | undefined
+    ) => void;
+    options: IChoiceGroupOption[];
+    defaultSelectedKey?: string | number;
+}
+
+/**
+ * Custom styled wrapper for default fluentui component
+ */
+export default function BaseCheckbox(props: Props) {
```

Comment:
Component name doesn't match file
```suggestion
export default function ChoiceGroup(props: Props) {
```